### DISCUSSION TITLE
Alternative to bank sync to import transactions in real time

### DISCUSSION THREAD

**Discussion by rtrind** [actualbudget/releases#29] [Feature ideas]
First of all, I really appreciate your product and effort and I hope you are able to handle the influx of people leaving YNAB en masse, since your product really deserves it. One of the best alternatives in the zero based budget category.

I know a lot of people like the automatic transactions from bank sync, but my country is not supported most of the time and it seems this type of integration always go sideways, making it unreliable and expensive to support.

One alternative for me (and maybe this works for other people) is having some kind of mechanism to import transactions from SMS or mobile notifications. A parser of some kind is needed, but it could give the flexibility of allowing quick transaction entries for those of us who can't or won't ever use bank sync. 

YNAB does not have this, but since it has a REST API, it is relatively ease to make the parser myself and add the transaction semi-automatically. Your app already have an API, but since it depends on the app running and it uses a node structure, it's not exactly mobile friendly.

Any way to reach this, either with a revamped API available (which seems to be on the roadmap) or a way to directly explore the mobile notification/SMS system would be greatly appreciated and it's the killer feature for me to migrate to Actual.

Thanks a lot and congratulations for this awesome tool!

**Comment by bdoherty** [comment]
Cool idea.  It made me think about writing a tool that parsed email notifications send from my bank to my gmail account, however my silly bank only sends notifications about money in/out of my transaction account, and not my credit card (where I make 95% of my transactions).
